apps/opik-backend/src/main/resources/liquibase/db-app-analytics/migrations/000006_add_prompt_id_to_experiments.sql

```diff
@@ -0,0 +1,7 @@
+--liquibase formatted sql
+--changeset thiagohora:add_prompt_version_to_experiments
+
+ALTER TABLE ${ANALYTICS_DB_DATABASE_NAME}.experiments
+    ADD COLUMN IF NOT EXISTS prompt_id Nullable(FixedString(36)) DEFAULT NULL;
```

Comment:
These nullable fields aren't great from performance perspective. Since the FixedString performance hit isn't always that great over `String`, it might be preferable to define the later here and use the empty string to represent the absence of the value.

Anyway, not a blocker.